devicetypes/smartthings/zwave-motion-sensor.src/zwave-motion-sensor.groovy

```diff
@@ -88,6 +94,7 @@ def sensorValueEvent(value) {
 	if (value) {
 		createEvent(name: "motion", value: "active", descriptionText: "$device.displayName detected motion")
 	} else {
+		createEvent(name: "tamper", value: "clear")
```

Comment:
Unfortunately this won't work when the DTH is locally executing. Take a look at https://github.com/SmartThingsCommunity/SmartThingsPublic/blob/master/devicetypes/smartthings/aeon-multisensor-6.src/aeon-multisensor-6.groovy#L324 to see how we handled this for the Aeon Multisensor 6. Hopefully the same will work here.

Reply:
thanks. forgot about that.

Fixed*